examples/src/multi-file/endpoints/get-users.ts

```diff
@@ -0,0 +1,17 @@
+import {api, endpoint, Optional, queryParam} from "@zenclabs/spot";
+
+@api()
+export class Api {
+  @endpoint({
+    method: "GET",
+    path: "/users"
+  })
+  getUsers(
+    @queryParam({required: true}) limit: number
```

Comment:
Instead of having a `required` boolean, why not rely on TypeScript optional semantics?

In this case, you would use:
```@queryParam() limit: number```

If you wanted it to be optional, you could use either:
```
@queryParam() limit?: number;
@queryParam() limit: Optional<number>;
```

Reply:
changed to `@queryParam limit:Optional<number>`